test/app/setup.js

```diff
@@ -15,3 +15,13 @@ for (var i = 0;  i < 100; i++) {
   app.appendChild(el);
 }
 
+var root = document.getElementById('root');
+for (var i = 0;  i < 5; i++) {
```

Comment:
We should have more elements so you can scroll the 3rd element completely out of the root viewport